### PR TITLE
API Changes - May 2016

### PR DESCRIPTION
**Pull request by jamesmunns**
Current found changes. May require more in the near future.

### CODE REVIEW

**Comment by jamesmunns** [comment]
Apologies for the removal of trailing whitespace messing with the diff.

File: relayr/api.py [hunk 1]
```diff
@@ -1197,19 +1147,6 @@ def delete_device(self, deviceID):
         _, data = self.perform_request('DELETE', url, headers=self.headers)
         return data
 
-    def get_device_apps(self, deviceID):
```
Comment (jamesmunns) [inline relayr/api.py]: @theomega Did we also remove the POST and DELETE endpoints for `devices/{}/apps/{}`? The changelog only mentions GET. see `post_device_app()` and `delete_device_app()` in this file.

**Comment by theomega** [comment]
Those endpoints (POST and DELETE) didn't exist for a year, so please also remove them, they have never worked

**Comment by theomega** [comment]
Also, we need to change the code a little for the model: The `model` property on the device will also go away (it is deprecated). Only the `modelId` property of the device will stay. Currently `model` and `modelId` will contain the same

File: relayr/api.py [hunk 2]
```diff
@@ -1305,37 +1305,6 @@ def post_device_data(self, deviceID, data):
         _, data = self.perform_request('POST', url, data=data, headers=self.headers)
         return data
 
-    def post_device_app(self, deviceID, appID):
```
Comment (jamesmunns) [inline relayr/api.py]: @theomega Thanks for the heads up. Let me know if you see anything else out of place.

**Pull request merged**